android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="com_auth0_domain">dev-bajcmartinez.eu.auth0.com</string>
```

Comment:
Is it necessary to include the domain of an actual tenant (instead of a placeholder for the developer to replace) in the strings file?

Reply:
No, it is not at all necessary, I'm replacing it with placeholders. Thanks for the catch!